### PR TITLE
ChaiShi/AuthenticationToken

### PR DESCRIPTION
**Pull request by changchaishi**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ marshmallow = "==2.19.5"
 marshmallow-sqlalchemy = "==0.19.0"
 flask-migrate = "==2.5.2"
 flask-script = "==2.0.6"
+itsdangerous = "*"
```
Comment (Sirius207) [inline Pipfile:25]: Which version do you use?
You can replace '*' with specific version number

File: pt/config/__init__.py [hunk 2]
```diff
@@ -41,7 +41,7 @@
 # random
 RANDOM_GENERATOR = Random.new().read
 
-PEM_PATH = os.environ.get("PEM_PATH", "/home/energy-trading-platform/pt/rsa")
+PEM_PATH = os.environ.get("PEM_PATH", "/home/changchaishi/Desktop/energy-trading-platform/pt/rsa")
 # decrypt
```
Comment (Sirius207) [inline pt/config/__init__.py]: OK, we just remove the default path.

File: Pipfile (hunk 1 above)
Comment (changchaishi) [reply Pipfile:25]: fixed

File: pt/config/__init__.py (hunk 2 above)
Comment (changchaishi) [reply pt/config/__init__.py]: fixed: back to original path

File: pt/config/__init__.py [hunk 3]
```diff
@@ -53,3 +53,5 @@
     open(os.path.join(PEM_PATH, "ami_rsa_public.pem")).read()
 )
 PLAT_SIGNER = Signature_pkcs1_v1_5.new(AMI_RSA_PUB_KEY)
+
+APP_SECRET_KEY = os.environ.get('APP_SECRET_KEY', 'VCn>ZaI],B/K-rPtq|2eB^gps~H>45')
```
Comment (Sirius207) [inline pt/config/__init__.py:58]: Maybe We can add a comment to this app_secret_key

Comment (changchaishi) [inline pt/config/__init__.py:58]: the secret key to form the unique Serializer for short-lived token

**Review by Sirius207 — APPROVED** [review]
Finished

**Pull request merged**